SIGs/TWI/TWI Charter.md

```diff
@@ -0,0 +1,90 @@
+
+1. The Workload is confidentiality- and integrity-isolated from its hosting environment as well as peer workloads throughout the lifetime of the Workload instance
+2. Any credential representing a Workload Identity is bound to that Workload instance
+3. A Workload Credential can be traced back to the Workload’s Provenance and the credential issuer
+
+Practical (deployable, performant, compatible, manageable) solutions in the TWI space will have to satisfy several additional requirements, captured in **\[2]**.
+
+## Goals
+We envision ubiquitous use of Trustworthy Workload Identities as the preferred mechanism by which any two workloads can establish trust in each other.
+
+The goals of the TWI SIG are as follows:
+
+1. Precisely define and publish (in **\[2]**) all scenarios and requirements that Trustworthy Workload Identity must meet
+2. Create a TWI Reference Architecture that addresses all the scenarios and requirements documented in **\[2]**
+3. Achieve integration (“impedance match”) of the TWI Reference Architecture with applicable existing and evolving frameworks, standards and solutions
+4. Influence the evolution of the broader Workload Identity solution space as appropriate to ensure that TWI offers a compatible and attractive option
+5. Provide reference open source implementations adhering to the Reference Architecture to assist the TWI community
+6. Provide a forum for ongoing discussions on TWI topics among members of the Consortium as well as third parties
+
+Importantly, there is also a range of outcomes our work will seek to prevent:
+
+1. Fragmentation of the solution landscape into multiple incompatible or inconsistent approaches
+2. Gaps in the availability of solutions which could cause barriers to broad adoption
+3. Conflicts with the decisions made inside other CCC working groups (e.g., Attestation SIG, GRC SIG)
+
+To summarize, the overarching goal we must achieve is turning TWI into the [Schelling Point](https://en.wikipedia.org/wiki/Focal_point_\(game_theory\)) of Confidential Computing deployment, both for new and existing applications.
+
+## References
+1. TODO: Document summarizing current regulatory landscape
+2. Strong Workload Identity Scenarios and Requirements: <https://github.com/CCC-Attestation/meetings/blob/main/musings/workload-identity.md>
+3. Confidential Payload Governance pattern: <https://drive.google.com/drive/u/0/folders/1EaXIm1jK3af_oUG7lTLYL9QcS9BRO5gy>
+
+## Notes & Clarifications
+### The Deeper Meaning of TWI Definition
+Here we reiterate the TWI Definition and provide additional context and clarification.
+
+First, a couple of quick notes on supporting definitions.
+
+**Workload** as used in this document restricts the definition of the same term by WIMSE – “a running instance of software executing for a specific purpose” – to just that part of the code and configuration of the (WIMSE-defined) workload that is subject to Remote Attestation.
```

Comment:
Please add reference to WIMSE on line 65

Reply:
Done